src/inference/src/core.cpp

```diff
@@ -49,9 +49,13 @@ Core::Core(const std::string& xmlConfigFile) {
     _impl = std::make_shared<Impl>();
 
 #ifdef OPENVINO_STATIC_LIBRARY
-    _impl->register_plugins_in_registry(::getStaticPluginsRegistry());
+    OV_CORE_CALL_STATEMENT(_impl->register_plugins_in_registry(::getStaticPluginsRegistry());)
 #else
-    register_plugins(findPluginXML(xmlConfigFile));
+    OV_CORE_CALL_STATEMENT({
+        // If XML is default, load default plugins by absolute paths
+        auto loadByAbsPath = xmlConfigFile.empty();
+        _impl->register_plugins_in_registry(findPluginXML(xmlConfigFile), loadByAbsPath);
```

Comment:
Nitpick: just to avoid introduction of additional local variables
```suggestion
    OV_CORE_CALL_STATEMENT({
        // If XML is default, load default plugins by absolute paths
        _impl->register_plugins_in_registry(findPluginXML(xmlConfigFile), xmlConfigFile.empty());
```

Reply:
Is it readable/clear enough that `xmlConfigFile.empty()` means plugins will be loaded by absolute paths only?

Reply:
I will do it in next PR